pyproject.toml

```diff
@@ -1,25 +1,21 @@
 [build-system]
-requires = ["setuptools", "setuptools-scm[toml]", "wheel"]
+requires = ["setuptools", "wheel"]
 build-backend = "setuptools.build_meta"
 
 [project]
 name = "google_benchmark"
 description = "A library to benchmark code snippets."
-requires-python = ">=3.8"
-license = {file = "LICENSE"}
+requires-python = ">=3.10"
+license = { file = "LICENSE" }
 keywords = ["benchmark"]
 
-authors = [
-    {name = "Google", email = "benchmark-discuss@googlegroups.com"},
-]
+authors = [{ name = "Google", email = "benchmark-discuss@googlegroups.com" }]
 
 classifiers = [
     "Development Status :: 4 - Beta",
     "Intended Audience :: Developers",
     "Intended Audience :: Science/Research",
     "License :: OSI Approved :: Apache Software License",
-    "Programming Language :: Python :: 3.8",
```

Comment:
why are we dropping 3.8 and 3.9?

Reply:
I was mirroring NumPy's support policy of "every Python released in the previous 42 months", which is also JAX's policy (https://jax.readthedocs.io/en/latest/deprecation.html#python-and-numpy-version-support-policy). If we go for "everything that is not officially EOL" instead, I'll have to add it back.

Reply:
i think it's fine as long as we document it: https://google.github.io/benchmark/dependencies.html

Reply:
Sweet, I'll do that right quick.

Reply:
Done.